app/bootstrap.php

```diff
@@ -38,7 +38,5 @@
 	define('CURRENT_DIR', getcwd());
 }
 
-session_start();
```

Comment:
@LukasReschke please leave this in place. I use session to store info between ajax requests. :)